### ISSUE TITLE
[Bug] train_test_split() missing random_state in multiple projects — non-reproducible results

### ISSUE THREAD

**Issue by Daksh7785** [Niketkumardheeryan/ML-CaPsule#1575]
## 🐛 Bug Report

**Description:**
Several projects call `train_test_split()` without setting `random_state`. Every run
produces a different split, meaning accuracy metrics change on every execution. This is
especially harmful in a learning repo where students expect consistent, reproducible results.

## Affected Files

| File | Line |
|------|------|
| `Bitcoin Price Prediction Web App/app.py` | 50 |
| `Chicken_Disease_Classification/research/mlflow_exp/example.py` | 47 |
| `Social Media Fake Accounts Detection/jserver/model.py` | 76 |
| `Ensemble Methds in ML/voting.py`, `blending.py`, `stacking.py` | various |

## Current Code (Buggy)
```python
xtrain, xtest, ytrain, ytest = train_test_split(X, Y, test_size=0.2)
```

## Fix
```python
xtrain, xtest, ytrain, ytest = train_test_split(X, Y, test_size=0.2, random_state=42)
```

**Severity:** 🟡 Medium — produces non-reproducible results; misleads learners

**Comment by premdeshmane-01** [comment]
Hi @Daksh7785,

I would like to work on this issue under GSSoC 2026.

I can update the affected files by adding proper `random_state` values to `train_test_split()` calls to ensure reproducible and consistent results across runs.

Please assign this issue to me. Thank you!

**Comment by 1stAceofSpade** [comment]
Hi, I would like to work on this issue under GSSoC 2026.

I can update the affected files by adding proper `random_state` values to `train_test_split()` to ensure reproducible and consistent results.

Please assign this issue to me. Thank you!

**Comment by 1stAceofSpade** [comment]
Trash repo
